### PR TITLE
feat: devcard v2 styles

### PR DESCRIPTION
**Pull request by sshanzel**
Just missing the links, once uploaded, I will further test the feature.

### CODE REVIEW

**Review by idoshamun — CHANGES_REQUESTED** [review]
We don't have to support two versions here.
It's enough to support only the latest one.
Either way it will only affect the webapp which is always up to date

**Comment by sshanzel** [comment]
Ah, good point. Let me refactor it.

**Review by rebelchris — APPROVED** [review]
Nice work! Once we have the images good to go 🥳

File: src/templates/devcard.ts [hunk 1]
```diff
@@ -48,6 +63,8 @@ const rankShapes = [
                             <path d="M68,44.8472102 L68,65.1666667 L47.6805435,65.1666667 C47.6805435,63.737802 48.2176078,62.3089373 49.2917364,61.2088463 L64.0402102,46.4565435 L64.5269032,46.0355363 C65.5457706,45.2433189 66.7728853,44.8472102 68,44.8472102 Z" id="Blink" fill-opacity="0.64" fill="#FFFFFF"></path>`,
 ];
 
+const rankShapesV2 = [`iron`, ...rankShapes, 'legendar'];
```
[the remark this answers is not in the document]
Comment (sshanzel) [reply src/templates/devcard.ts]: Have updated it now to the actual svg paths 🚀

**Comment by sshanzel** [comment]
Already have them the expected name for the URL, so everything should be cool now. Will merge this before we merge the gamification since there will be a mismatch with the icons if this gets merged early.

**Pull request merged**